redfish-core/lib/hypervisor_system.hpp

```diff
@@ -585,6 +585,12 @@ inline void
                                             "hypervisor", "EthernetInterfaces",
                                             ifaceId));
             }
+            else if (ec == boost::system::errc::io_error)
+            {
+                messages::propertyValueFormatError(
+                    asyncResp->res, "Address/PrefixLength",
```

Comment:
Are you sure we can give 2 properties like this?
Prefix length is an integer value. There is no format for that as IPv6 addresses.

Reply:
We can give the whole json data here instead. But I checked giving an invalid prefix length and it failed with a different error itself - invalid value for prefix length. So we are good if we just give address here